storage/collections/ibft_storage.go

```diff
@@ -70,7 +74,7 @@ func (i *IbftStorage) GetCurrentInstance(pk []byte) (*proto.State, error) {
 func (i *IbftStorage) SaveDecided(signedMsg *proto.SignedMessage) error {
 	value, err := json.Marshal(signedMsg)
 	if err != nil {
-		i.logger.Error("failed serializing decided msg", zap.Error(err))
+		return err
```

Comment:
its better to use errors (for stack tracing)

Reply:
what you mean? wrap the error?

Reply:
errors.New()